.gitignore

```diff
@@ -17,3 +17,4 @@
 /yarn-error.log
 
 .byebug_history
+coverage/
```

Comment:
Added this to ignore...rolling with the modified of the reports we already committed.